### ISSUE TITLE
wit.ai is failing

### ISSUE THREAD

**Issue by brendanclement** [brendanclement/XRHackathon#3]
When i speak to one of the characters - i immediately hear a response with a "what are you talking about?" indicating an error.

going to pull a stack trace from the device

**Comment by brendanclement** [comment]
The logs show that no intent is being sent back from wit.ai

```
11-19 20:34:51.460 20442 20589 I GuardianInput: RightTrigger up
11-19 20:34:51.468 10001 10022 I Unity   : [Mic] Stopped recording with Android audio input
11-19 20:34:51.468 10001 10022 I Unity   : Facebook.WitAi.Lib.Mic:StopRecording()
11-19 20:34:51.468 10001 10022 I Unity   : Facebook.WitAi.Wit:DeactivateRequest()
11-19 20:34:51.468 10001 10022 I Unity   : Facebook.WitAi.Wit:Deactivate()
11-19 20:34:51.468 10001 10022 I Unity   : UnityEngine.Events.UnityAction:Invoke()
11-19 20:34:51.468 10001 10022 I Unity   : UnityEngine.Events.UnityEvent`1:Invoke(T0)
11-19 20:34:51.468 10001 10022 I Unity   : UnityEngine.XR.Interaction.Toolkit.XRBaseInteractable:OnDeactivated(DeactivateEventArgs)
11-19 20:34:51.468 10001 10022 I Unity   : UnityEngine.XR.Interaction.Toolkit.XRRayInteractor:ProcessInteractor(UpdatePhase)
11-19 20:34:51.468 10001 10022 I Unity   : UnityEngine.XR.Interaction.Toolkit.XRInteractionManager:ProcessInteractors(UpdatePhase)
11-19 20:34:51.468 10001 10022 I Unity   : UnityEngine.XR.Interaction.Toolkit.XRInteractionManager:Update()
11-19 20:34:51.468 10001 10022 I Unity   : 
11-19 20:34:51.508 10001 10022 I OpenXR_Actions: -------- xrApplyHapticFeedback ---------
11-19 20:34:51.593 10001 10022 E Unity   : GAME: Got intent {"entities":{}, "intents":[  ], "traits":{}}
11-19 20:34:51.593 10001 10022 E Unity   : NPCSpeak:Speak(WitResponseNode)
11-19 20:34:51.593 10001 10022 E Unity   : UnityEngine.Events.UnityAction`1:Invoke(T0)
11-19 20:34:51.593 10001 10022 E Unity   : UnityEngine.Events.UnityEvent`1:Invoke(T0)
11-19 20:34:51.593 10001 10022 E Unity   : Facebook.WitAi.Wit:HandleResult(WitRequest)
11-19 20:34:51.593 10001 10022 E Unity   : System.Action:Invoke()
11-19 20:34:51.593 10001 10022 E Unity   : Facebook.WitAi.Wit:Update()
11-19 20:34:51.593 10001 10022 E Unity   : 
```

**Comment by brendanclement** [comment]
when i revert to 8826db7147e2c3e961be0abe861e95f41bb3d29a the voice works again

**Comment by brendanclement** [comment]
the issue seems to resolve itself when there is only one AppVoiceExperience script in the scene. i removed all but george's and i'm able to talk to him now

**Comment by brendanclement** [comment]
i fixed it by adding one voiceappexperience and another script to track which npc the player is currently interacting with

**Comment by brendanclement** [comment]
fixed in 90d35900548997591280ada9e1dda8dbfeeffaaa
